google/cloud/firestore_v1/async_aggregation.py

```diff
@@ -124,14 +153,21 @@ async def _make_stream(
 
         async for response in response_iterator:
             result = _query_response_to_result(response)
-            yield result
+            if result:
```

Comment:
Note: I added this so it doesn't yield an empty list when we have `explain_options=ExplainOptions(analyze=False)`. In this case no results are returned but we get a message containing explain metrics info.